src/NewsService.Models.Broker/Responses/IGetDepartmentResponse.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace LT.DigitalOffice.Broker.Responses
+{
+    public interface IGetDepartmentResponse
```

Comment:
[AutoInjectRequest(nameof(RabbitMqConfig ...

Reply:
It's not request